main/default.cfg

```diff
@@ -7,45 +7,33 @@ unbindall
 //
 // WEAPONS/UPGRADES
 //
+
 bind  MOUSE1        +attack
 bind  MOUSE2        +attack2
 
-bind  [             weapprev
-bind  ]             weapnext
-bind  mwheelup      weapprev
-bind  mwheeldown    weapnext
+bind  mwheelup      "weapprev; +useitem"
```

Comment:
Doesn't quite work for us, as grenade is a usable right now. Calling +useitem on it will throw it upon switching.